pages/dkp/kaptain/1.3.0/user-management/index.md

```diff
@@ -32,20 +34,33 @@ Predefined `ClusterRoles` for Kubeflow tenants are as follows:
 For a chart of the permissions granted to each `ClusterRole`, see the [Permissions Charts](#permissions-charts).
 
 ## Onboarding new users
+
 ### Overview
+
 Kaptain provides an abstraction called `Profile` which is assigned to a user and bound to a namespace along with additional
 resources such as namespace-scoped service accounts, RBAC `RoleBinding`s, Istio `ServiceRole` and `ServiceRoleBinding`.
 Kubeflow grants users with namespace admin permissions for their namespaces.
 
 ### Automatic profile creation
-When an authenticated user logs into the system and visits the central dashboard for the first time, they trigger a profile creation automatically, this is referred to as a "Registration Flow".
-Automatic profile creation is disabled by default. To enable it, set the `registrationFlow` parameter to `true`:
 
-```
-kubectl kudo install --instance kaptain --namespace kubeflow --create-namespace ./kubeflow-1.4.0_1.3.0.tgz -p registrationFlow=true
-```
```

Comment:
I think it still might be a good idea to show how to enable it during the fresh installation. Should we keep this command in addition to `kubectl kudo update`?